File: sync/rest/map-permissions/update-permission/update-permission.5.x.cs
// Download the twilio-csharp library from twilio.com/docs/libraries/csharp
using System;
using Newtonsoft.Json;
using Twilio;
using Twilio.Exceptions;
using Twilio.Rest.Sync.V1.Service.SyncMap;

public class Example
{
    public static void Main(string[] args)
    {
        // Find your Account SID and Auth Token at twilio.com/console
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";
        const string serviceSid = "ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";

        TwilioClient.Init(accountSid, authToken);

        try
        {
            SyncMapPermissionResource.Update(serviceSid, "Players", "bob", true, true, true);
        }
        catch (ApiException err)
        {
            Console.WriteLine(err.Message);
        }
    }
}

File: sync/rest/map-permissions/update-permission/update-permission.5.x.rb
require('twilio-ruby')

account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

client = Twilio::REST::Client.new(account_sid, auth_token)
service = client.sync.v1.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')

begin
  response = service.sync_maps('Players')
                    .sync_map_permissions('bob')
                    .update(read: true, write: true, manage: true)
  puts response
rescue Twilio::REST::TwilioError => err
  puts err
end
